test/test-app/src/app/app.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
```

Comment:
app_component.ts

Reply:
@jiady Is there a reason why we want to use underscores for file names over hyphens? I thought that convention was to do `app-component.ts`